doc/_includes/inverse.rst

```diff
@@ -156,8 +156,8 @@ distribution.
 
 A convenient choice for the source-covariance matrix :math:`R` is such that
 :math:`\text{trace}(\tilde{G} R \tilde{G}^\top) / \text{trace}(I) = 1`. With this
-choice we can approximate :math:`\lambda^2 \sim 1/SNR`, where SNR is the
-(power) signal-to-noise ratio of the whitened data.
+choice we can approximate :math:`\lambda^2 \sim 1/rm{SNR}^2`, where SNR is the
```

Comment:
I think there's a missing backslash before `rm` here

https://40144-1301584-gh.circle-artifacts.com/0/dev/overview/implementation.html?highlight=convenient%20choice#whitening-and-scaling

Reply:
dang, I was 1 minute too slow

Reply:
Will push to `main`